homeassistant/components/songpal/config_flow.py

```diff
@@ -21,7 +21,7 @@
 class SongpalConfig:
     """Device Configuration."""
 
-    def __init__(self, name: str, host: str | None, endpoint: str) -> None:
+    def __init__(self, name: str, host: str, endpoint: str) -> None:
```

Comment:
It looks to me like `urlparse` types it as optional, but in a real scenario it would always be present.